### PR TITLE
docs: add bi-weekly-report (9.11-9.24)

### PR DESCRIPTION
**Pull request by saltism**
Add: bi-weekly report (9.11-9.24)

### CODE REVIEW

File: blog/en/blog/2023/09/26/bi-weekly report.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/09/25/Ttkq3Ecr_%E7%A4%BE%E5%8C%BA%E5%8F%8C%E5%91%A8%E6%8A%A5-%E5%85%AC%E4%BC%97%E5%8F%B7%E5%A4%B4%E5%9B%BE-%E8%8B%B1%E6%96%87.png
+---
+
+> We have recently made fixes and improvements to certain features of Apache APISIX. These include utilizing xrpc to support the dubbo protocol, enabling the recording of OpenTelemetry variables in the access_log, and allowing modification of request bodies in external plugins. For more details, please refer to the biweekly report.
```
Comment (Yilialinn) [inline blog/en/blog/2023/09/26/bi-weekly report.md]:
```suggestion
> We have recently made fixes and improvements to certain features of Apache APISIX. These include utilizing `xrpc` to support the dubbo protocol, enabling the recording of OpenTelemetry variables in the `access_log`, and allowing modification of request bodies in external plugins. For more details, please refer to the biweekly report.
```

File: blog/en/blog/2023/09/26/bi-weekly report.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+From 9.11 to 9.24, 23 contributors submitted 43 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+
+We have recently fixed and improved some features, and the summary of the updates is as follows:
+
+1. Utilize xrpc to support the dubbo protocol
+
+2. Enable the recording of OpenTelemetry variables in the access_log
```
Comment (Yilialinn) [inline blog/en/blog/2023/09/26/bi-weekly report.md]:
```suggestion
1. Utilize `xrpc` to support the dubbo protocol

2. Enable the recording of OpenTelemetry variables in the `access_log`
```

File: blog/zh/blog/2023/09/26/bi-weekly report.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+从 2023.9.11 - 2023.9.24，有 23 位开发者为 Apache APISIX 提交了 43 个 commit。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
+
+我们近期对部分功能进行了修复或改善，更新内容汇总如下：
+
+1. 使用 xrpc 支持 dubbo 协议
+
+2. 支持在 access_log 中记录 opentelemetry 变量
```
Comment (Yilialinn) [inline blog/zh/blog/2023/09/26/bi-weekly report.md]:
```suggestion
1. 使用 `xrpc` 支持 dubbo 协议

2. 支持在 `access_log` 中记录 opentelemetry 变量
```

File: blog/zh/blog/2023/09/26/bi-weekly report.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- Apache APISIX
+- API 网关
+- 社区周报
+- 贡献者
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/09/25/FIgUq7x3_%E7%A4%BE%E5%8C%BA%E5%8F%8C%E5%91%A8%E6%8A%A5-%E5%85%AC%E4%BC%97%E5%8F%B7%E5%A4%B4%E5%9B%BE-%E4%B8%AD%E6%96%87.png
+---
+
+> 我们近期对 Apache APISIX 的部分功能进行了修复或改善，包括使用 xrpc 支持 dubbo 协议，支持在 access_log 中记录 opentelemetry 变量，以及支持在外部插件中修改请求 body。详情请阅读本期双周报。
```
Comment (Yilialinn) [inline blog/zh/blog/2023/09/26/bi-weekly report.md]:
```suggestion
> 我们近期对 Apache APISIX 的部分功能进行了修复或改善，包括使用 `xrpc` 支持 dubbo 协议，支持在 `access_log` 中记录 opentelemetry 变量，以及支持在外部插件中修改请求 body。详情请阅读本期双周报。
```

**Comment by yzeng25** [comment]
Please rename both files' names to `bi-weekly-report`, i.e. `blog/en/blog/2023/09/26/bi-weekly-report.md`; `blog/zh/blog/2023/09/26/bi-weekly-report.md`.
image

**Review by yzeng25 — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Review by Yilialinn — APPROVED** [review]

**Pull request merged**